### ISSUE TITLE
Client Notes

### ISSUE THREAD

**Issue by Justin-Schneider** [Justin-Schneider/HealthApp#1]
These are the notes I took during the meeting, they aren't super clear, so feel free to ask for clarification.

Notes
- Leader board
- monthly notification of progress
- #c62828 #00838f (Color Scheme)

Login 
- Standard stuff

Registration 
- Standard stuff 
- As well as a group (I thought it best to have a dropdown to select a group, and if you didn't see a group you wanted to join you should be able to make one)

Home
- slides that show heart rate, steps, calories
- https://ionicframework.com/docs/components/#slides 

Steps
- top information about steps
- bottom line graph about weekly steps taken

Calories
- top input form for calories, name
- middle graph
- bottom scrolling card for what you ate that day breakfast, lunch, and dinner

Hydration
- how many glasses of water did you drink 
- water bottle on page that fills up based on your hydration 

Heart Rate
- heart in the middle of the screen showing there heart rate in the middle

LeaderBoard 
- Showing who is doing the best in some categories
